src/components/ErrorBanner/ErrorBanner.html

```diff
@@ -0,0 +1,20 @@
+<!-- Copyright (c) Microsoft. All rights reserved. Licensed under the MIT license. See LICENSE in the project root for license information. -->
+
+<div class="ms-ErrorBanner">
+  <div class="ms-ErrorBanner-content">
+    <div class="ms-ErrorBanner-text">
+      <div class="ms-ErrorBanner-clipper">
+        You've reached your total storage on OneDrive. Please upgrade your storage plan if you need more storage.
+      </div>
+    </div>
+    <button class="ms-ErrorBanner-expand">
+      <i class="ms-Icon ms-Icon--chevronsDown"></i>
+    </button>
+    <div class="ms-ErrorBanner-action">
+      <button class="ms-Button ms-Button--primary ms-fontColor-neutralLight">Get More Storage</button>
+    </div>
+  </div>
+  <div class="ms-ErrorBanner-close">
```

Comment:
This should probably be a button so that it's keyboard accessible like the ms-ErrorBanner-expand button.